addons/udes_stock/models/stock_picking.py

```diff
@@ -235,43 +249,6 @@ def get_move_lines(self, done=None):
             return self.move_line_ids.get_lines_incomplete()
         return mls
 
-    def _backorder_move_lines(self, mls=None):
```

Comment:
I don't think we should be deleting this function, only reversing which move lines we move into the backorder.

Reply:
I thought it might be nice just to rely on action_done, so we have one behaviour.  I don't see how we would want to split one set of move lines into one picking then another, because i guess refactoring should do that?